tests/pyccel/wrap_scripts/c_tests/class_renaming.c

```diff
@@ -0,0 +1,20 @@
+#include "class_renaming.h"
+
+
+/*........................................*/
+void Counter__create(struct class_renaming__Counter* self, int64_t start)
+{
+    self->value = start;
+}
+/*........................................*/
+/*........................................*/
+void Counter__free(const struct class_renaming__Counter* self)
+{
+}
```

Comment:
The Fortran version of this file sets the counter value to `-1`:

```suggestion
void Counter__free(const struct class_renaming__Counter* self)
{
    self->value = INT64_C(-1);
}
```